### ISSUE TITLE
Two h1 tags on each HTML page

### ISSUE THREAD

**Issue by barbara-sfx** [bitprophet/alabaster#137]
tl;dr I have posted a query in the sphinx discussion group (https://groups.google.com/forum/#!topic/sphinx-users/AQ9VbJwbXT8)  and the most recent response is:

I think it is a bug of alabaster. "about" sidebar uses `<h1>` tag, but
it should be used in main content.
https://github.com/bitprophet/alabaster/blob/master/alabaster/about.html#L6

=============

Here is the Issue:

SEO folks asking me for more fixes. By default, my pages wind up with two H1 tags, which apparently google hates. In a typical page, they look like this:

```
<h1>Dashboards<a class="headerlink" href="#dashboards" title="Permalink to this headline">¶</a></h1>

<h1 class="logo"><a href="../index.html">SignalFx</a></h1>

```
I can't find where the class for logo is designed to be in an H1 tag, or even what this link refers to. It's identical in every file, no matter where it is in the toctree hierarchy, and i don't see a link to SignalFx anywhere in the docs that always goes to ../index.html. So I can't figure out what this is or where it's coming from. 

We are using a customized alabaster theme with a few custom extensions (open link on a new page, show/hide text under a heading). I compared our theme to the default theme and the only changes are related to fonts and spacing. 

Actually I just did a build with the default theme and ran into the same issue.

===========

The most recent response is:

I think it is a bug of alabaster. "about" sidebar uses `<h1> `tag, but
it should be used in main content.
https://github.com/bitprophet/alabaster/blob/master/alabaster/about.html#L6

===========

Can you confirm if this is a bug, and/or tell me if there is anything I can do to stop getting two H1s in every page? Our docs are at docs.signalfx.com if you need to look at the source for a file.

Thanks -- Barbara

**Comment by barbara-sfx** [comment]
@bitprophet Hi - can you tell me if adding this to a milestone means fixing/addressing it is on a to-do list for an upcoming release?

**Comment by bitprophet** [comment]
@barbara-sfx Correct! I can't promise when that'll be but it means it is on the roadmap.

**Comment by zabbir007** [comment]
uses <h1> tag
